vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -173,6 +168,10 @@ def save_guest_devices_inventory(hardware, hashes)
         # Save the hardware to force an id if not found
         hardware.save! if hardware.id.nil?
         h[:network][:hardware_id] = hardware.id
+      else
+        # ensure save_network_inventory gets called
+        # it will be deleted
+        h[:network] = {}
```

Comment:
> This seems wrong. If a key doesn't exist, it is ignored (i.e. NOT deleted).

Network doesn't follow this case. so that is the reasoning behind this PR.

I was able to find the code that set :network in the hash for rhevm and vc, but not for the others.